### PR TITLE
Privacy Policy

### PR DESCRIPTION
**Pull request by joshrsamuel**
# Description

**This PR brings the introduction of the long awaited privacy policy!** The privacy policy can be found by navigating to the new privacy page, where you will see a PDF embedded. This privacy policy was added because in order for the VR team to have their app accepted to the Meta Quest Store, our site needs a privacy policy.

This change in our codebase is coupled with an update to our Drupal. When visiting Drupal, you will notice the addition of a new content type, "PDFs". This was made to avoid storing the privacy policy locally, which isn't best practice. In the future, we could store other PDFs, or we could shift away from this method, but since the privacy policy was needed ASAP, I implemented it this way.

Task ID: 62 & 101

## Type of change

- [x] New feature ✨ (non-breaking change which adds functionality)
- [x] Drupal update 💧 (update to GT Drupal)

# Media

### Before
image

### After (ignore styling differences, as those will be resolved upon merge)
image

## Privacy Policy Page

image

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/components/privacyPolicy.js
Comment (AmaraWilliams) [inline src/components/privacyPolicy.js:1]: there may be a problem with this? I clicked on "privacy policy" in the footer and got this page:
image
The embedded pdf never popped up for me.

**Review by AmaraWilliams — COMMENTED** [review]
I'm unsure why it does not work for me so I will allow others to review and request changes in case it's a "me" problem

File: src/components/privacyPolicy.js
Comment (joshrsamuel) [reply src/components/privacyPolicy.js:1]: That's interesting. I tested the link in an incognito tab to make sure accessing the link wasn't dependent on logins/cookies. If others have the same issue, I can investigate it further.

**Review by Twallace40 — APPROVED** [review]
privacyPolicyScreenshot
The privacy policy shows up fine on my end, and all the hyperlinks within the PDF itself work as intended.

**Review by AmaraWilliams — APPROVED** [review]
The embedded pdf works on my end now (I think I forgot to git pull or something). The links work as expected. 
image

**Pull request merged**